src/main/java/zd/zero/waifu/motivator/plugin/assets/VisualMotivationAssetProvider.kt

```diff
@@ -27,8 +27,8 @@ object VisualMotivationAssetProvider {
         return MotivationAsset(
             textualAssetDefinition.title,
             """
-                 <img src='${visualAssetDefinition.imagePath}' alt='${visualAssetDefinition.imageAlt}' width='256' />
-                 <br><br><br><br><br><br><br><br><br><br>
```

Comment:
So I just removed the spacing between the text and image, as I find it has too much space. What do you think about the compact alert?

Reply:
I like it! I would just like something that consistently shows the images.